src/app/zap-templates/zcl/data-model/chip/keypad-input-cluster.xml

```diff
@@ -44,7 +44,7 @@ limitations under the License.
     <item name="InvalidKeyInCurrentState" value="0x02"/>
   </enum>
 
-  <enum name="CecKeyCode" type="enum8">
+  <enum name="CecKeyCodeEnum" type="enum8">
```

Comment:
If we're renaming things anyway, wouldn't "CECKeyCodeEnum" make more sense?  "CEC" is an acronym...

Reply:
Updated to CEC instead of Cec.